app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,7 @@
 class PagesController < ApplicationController
   def index
     @flight = Flight.new
```

Comment:
Yes, but in this case, I'm displaying a `form_for @flight` in the home page of the application - `PagesController#index`